context.go

```diff
@@ -684,15 +684,23 @@ func flipBoolValue(value reflect.Value) error {
 
 func (c *Context) parseFlag(flags []*Flag, match string) (err error) {
 	candidates := []string{}
+
+	newCandidate := func(s string) bool {
```

Comment:
Can you remove this please? Just inline the `candidates = append(candidates, s)`

Reply:
Sure, removed and inlined the logic